packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx

```diff
@@ -321,6 +326,27 @@ export class HorizontalBarChartBase extends React.Component<IHorizontalBarChartP
       startingPoint.push(prevPosition);
 
       const xValue = point.horizontalBarChartdata!.x;
+
+      if (index > 0 && this.props.variant === HorizontalBarChartVariant.Relative) {
```

Comment:
Add a name to index = 0 and index = 1 etc.

Reply:
done